.github/workflows/build-x64.yml

```diff
@@ -83,3 +83,8 @@ jobs:
         msbuild.exe /property:Platform=x64 /property:configuration="Release" src\JPEGView.sln
 
       shell: cmd
+    - name: Upload output
+      uses: actions/upload-artifact@v3
```

Comment:
@qbnu this new upload-artifact thing you added is AMAZING!  Now I can essentially test "official" nightlies without having to do a manual release process.  Thanks for all the cool additions you're bringing to this project!